includes/frontend.php

```diff
@@ -673,6 +673,15 @@ public function enqueue_styles() {
 				wp_enqueue_style( 'elementor-wp-admin-bar' );
 			}
 
+			if ( Plugin::$instance->experiments->is_feature_active( 'e_css_smooth_scroll' ) ) {
+				wp_enqueue_style(
```

Comment:
Should we not only register the style here, and enqueue it only when needed?

Reply:
@louiswol94 I am actually not sure what should be condition here. Only if an anchor is available? But we check that with JS. In that case we could load it with the `assets/dev/js/frontend/utils/assets-loader.js`.

Alternatively we could load it on every page load in one of the default frontend file (unconditionally). 

Wdyt?

Reply:
@louiswol94 Maybe this is fine actually.

Reply:
Do you mean we can always load it?

Reply:
@louiswol94 I think so.
It is just one line of css.